### ISSUE TITLE
help with classifier.predict and predicted_classes

### ISSUE THREAD

**Issue by IvanDeFalco** [tensorflow/tensorflow#13951]
### System information
- custom code: no, it is the one in https://www.tensorflow.org/get_started/estimator
- system: Apple
- OS: Mac OsX 10.13
- TensorFlow version: 1.3.0
- Python version: 3.6.3
- GPU model: AMD FirePro D700 (actually, two such GPUs)

### Describe the problem
Dear all,
I am running the simple iris program:
https://www.tensorflow.org/get_started/estimator
under python 3.6.3 and tensorflow 1.3.0.
The program executes correctly, apart from the very last part, i.e. the one related to the confusion matrix.
In fact, the result I get for the confusion matrix is:
New Samples, Class Predictions:    [array([b'1'], dtype=object), array([b'2'], dtype=object)]
rather than the expected output:
New Samples, Class Predictions:    [1 2]
Has anything about confusion matrix changed in the latest release?
If so, how should I modify that part of the code?
Thank you very much for your help!
Best regards
Ivan

### Source code / logs
https://www.tensorflow.org/get_started/estimator

**Comment by mcterrien** [comment]
Hi, I have the same issue with:
Windows 10 Pro, TensorFlow version: 1.3.0, Python version: 3.5.2
Any help is welcome. Thanks!

**Comment by nillpatel** [comment]
I have same issue with linux mint.

**Comment by ispirmustafa** [comment]
tf.estimator.DNNClassifier returns four predictions: logits, probabilities, class_ids, classes.
class_id is integer and classes is string
